test/governance/governed.test.ts

```diff
@@ -3,42 +3,46 @@ import hre from 'hardhat'
 import '@nomiclabs/hardhat-ethers'
 
 import { Governed } from '../../build/types/Governed'
-
-import { getAccounts, Account } from '../lib/testHelpers'
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers'
+import { helpers } from '@graphprotocol/sdk'
 
 const { ethers } = hre
 const { AddressZero } = ethers.constants
 
 describe('Governed', () => {
-  let me: Account
-  let governor: Account
+  const graph = hre.graph()
+  let me: SignerWithAddress
+  let governor: SignerWithAddress
 
   let governed: Governed
 
   beforeEach(async function () {
-    ;[me, governor] = await getAccounts()
+    ;[me, governor] = await graph.getTestAccounts()
+
+    await helpers.setAutoMine(true)
```

Comment:
What's the reason to add this here?

Reply:
The node would otherwise be set to not automine and be on a 13s block time (this defaults are defined in `hardhat.config.ts`. Previously the deployment/fixtures code was turning the automine on.

Now I'm not sure why the default config is to be on a 13s clock... for a local dev/test node I prefer a fast automine node, so maybe we should change that. Created https://github.com/graphprotocol/contracts/issues/886 to track this and investigate further.

Reply:
iirc the automine off default was made to support the local network setup (https://github.com/edgeandnode/local-network) but afaict it's not needed anymore (I see that repo actually sets automine to true).

So I think we should keep the previous behavior and turning this on in the fixture load function or even better in hardhat.config.ts, as the tests are designed to run with automine on (I suspect something is still turning on automine now? otherwise CI would take a lot longer to run?) and the 13s block time could cause race conditions. We used to see tests randomly failing because of this.

Reply:
Ok I've changed the `hardhat` network so that it automines by default. Everything seems to be working 💪